src/platform/Kconfig

```diff
@@ -339,11 +339,6 @@ config CAVS
 	bool
 	default n
 	select INTEL
-	select XT_INTERRUPT_LEVEL_2
-	select XT_INTERRUPT_LEVEL_5
-	select INTEL_MN
-	select WAKEUP_HOOK
-	select SCHEDULE_DMA_SINGLE_CHANNEL
```

Comment:
nice! Can some of these be removed completely? E.g. `INTEL_MN` looks Intel-specific, and we don't do XTOS builds on "main" any more?

Reply:
Others are removed, but INTEL_MN seems to have more complex baggage still in place.  I'm afraid the Zephyr SSP driver depends on this (still)... needs to be cleaned up before this can be merged.

Reply:
@lyakh ok, now filed https://github.com/zephyrproject-rtos/zephyr/pull/81734 for Zephyr and will submit a separate PR for SOF for this. I'll update this series to still keep INTEL_MN.